src/screens/SignUp.js

```diff
@@ -1,44 +1,69 @@
-import React, { useState } from 'react';
-import { View, Text, TextInput, TouchableOpacity, StyleSheet } from 'react-native';
-import { user } from '../../style/styles';
+import React, {useState} from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+import {user, welcome} from '../../style/styles';
 
 const SignUp = ({navigation}) => {
-  const [email, setEmail] = useState('');
+  const [name, setName] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
   const [password, setPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
 
   const handleSignUp = () => {
     // Handle the sign-up logic here
-    console.log('Signing up with:', email, password);
+    console.log('Signing up with:',name,  phoneNumber, password);
   };
 
   return (
     <View style={user.container}>
+       {/* <Image source={require('../../assets/images/leaves.jpg')} style={welcome.logo} /> */}
       <Text style={user.title}>Sign Up</Text>
 
       <TextInput
         style={user.input}
-        placeholder="Email"
-        value={email}
-        onChangeText={setEmail}
-        keyboardType="email-address"
-        autoCapitalize="none"
+        placeholder="Enter Your Name"
+        value={name}
+        onChangeText={text => setName(text)}
       />
-
       <TextInput
         style={user.input}
-        placeholder="Password"
-        value={password}
-        onChangeText={setPassword}
-        secureTextEntry
+        placeholder="Phone Number"
+        keyboardType="phone-pad"
+        value={phoneNumber}
+        onChangeText={text => setPhoneNumber(text)}
       />
 
+      <View style={user.passwordContainer}>
+        <TextInput
+          style={user.input}
+          placeholder="Password"
+          secureTextEntry={!showPassword}
+          value={password}
+          onChangeText={text => setPassword(text)}
+        />
+        {/* <TouchableOpacity onPress={() => setShowPassword(!showPassword)} style={styles.iconContainer}>
+          <Icon name={showPassword ? 'eye' : 'eye-slash'} size={20} color="gray" />
+        </TouchableOpacity> */}
+      </View>
+
       <TouchableOpacity style={user.button} onPress={handleSignUp}>
-        <Text style={user.buttonText}>Sign Up</Text>
-        {/* //adnan530@gmail.com addi3214 & adnan0@gmail.com kamranbhai */}
-      </TouchableOpacity> 
+        <Text style={user.ButtonText}>Sign Up</Text>
+      </TouchableOpacity>
+      <Text style={user.signUpLink}>
+        Already have an account?{' '}
+        <Text
+          style={user.clickHere}
+          onPress={() => navigation.navigate('signin')}>
+          Click here
+        </Text>
+      </Text>
     </View>
   );
 };
 
```

Comment:
are there any validations for form data? 🤐

Reply:
The answer is No. but no problem we can set the validations. please see 

1. react-hook-form
2. formik
3. js validation
4. react validations etc

Reply:
the answer is No. but no problem we have many things to do. please R&D these things:

- react-hook-form
- formik
- js validations etc
and we can also do this manually